profondo/cnn.py

```diff
@@ -151,5 +251,6 @@ def report_callback(epoch, model, x_val, y_val):
         callbacks=callbacks)
 
     # Load best model and evaluate it on the test data
-    model = load_model(x_test, y_test)
-    report_callback(0, model, x_test, y_test)
+    if exists(MODEL_FILENAME):
```

Comment:
If the model doesn't exist, `load_model` builds a new one from scratch. I'm happy to leave this as it is anyway, as long as we're aware of that :)